server/.env.example

```diff
@@ -7,4 +7,7 @@ CLIENT=http://localhost:1234
 
 GOOGLE_CLIENT_ID = **
 GOOGLE_CLIENT_SECRET = **
-SESSION_SECRET = keyboardcat
\ No newline at end of file
+SESSION_SECRET = keyboardcat
+
+EMAIL_ADD = *email addess to send notifications from*
```

Comment:
I would personally use EMAIL_ADDRESS to be explicit in this case but doesn't really matter